buildflow/core/app/runtime/actors/runtime.py

```diff
@@ -36,13 +38,23 @@ def as_dict(self):
         }
 
 
+@dataclasses.dataclass
+class ProcessPoolReference:
+    actor_handle: ActorHandle
+    processor: ProcessorAPI
+    options: ProcessorOptions
+
+
 @ray.remote(num_cpus=0.1)
 class RuntimeActor(Runtime):
     def __init__(
         self,
         run_id: RunID,
         *,
         runtime_options: RuntimeOptions,
+        # NOTE: we set this here instead of options since users don't need to set it.
+        # if we want we could expose it in options.
+        checkin_loop_frequency_sec: int = 5,
```

Comment:
Maybe this should be in runtime options, and then we could make runtime and infra options "private" and users would set any public fields in flow options?

Reply:
moved to runtime options